az-pipelines/entrypoint.sh

```diff
@@ -27,6 +31,38 @@ AZDEVOPS_URL="https://dev.azure.com/${AZURE_PIPELINE_ORGANIZATION}/"
 vsts configure --defaults instance=${AZDEVOPS_URL} project=${AZURE_PIPELINE_PROJECT}
     
 vsts login --token ${AZURE_PIPELINE_TOKEN}
-    
-echo "Queueing Azure pipeline: ${AZURE_PIPELINE_NAME}"
-vsts build queue --definition-name ${AZURE_PIPELINE_NAME}
\ No newline at end of file
+  
+PIPELINES=$( vsts build definition list --name ${AZURE_PIPELINE_NAME} )
+COUNT=$( echo ${PIPELINES} | jq length )
+
+if [ $COUNT -eq 0 ]; 
+then
+   echo "No build definition found with name: ${AZURE_PIPELINE_NAME}". >&2
+   exit 1;
+fi
+
+if [ $COUNT -gt 1 ]; 
+then
+    echo "Multple build definitions were found with name: ${AZURE_PIPELINE_NAME}. Pass unique build definition name and try again." >&2
+    exit 1;
+fi
+
+BUILD_DEFINITION_ID=$( echo ${PIPELINES} | jq .[0].id -r )
+BUILD_DEFINITION=$( vsts build definition show --id ${BUILD_DEFINITION_ID} )
+REPOSITORY_NAME=$( echo ${BUILD_DEFINITION} | jq .repository.name -r )
+REPOSITORY_TYPE=$( echo ${BUILD_DEFINITION} | jq .repository.type -r )
+
+if [ "$REPOSITORY_NAME" = "$GITHUB_REPOSITORY" ] && ["$REPOSITORY_TYPE" = "GitHub" ]; 
```

Comment:
Do case comparison

Reply:
I have verified. This is by default ignore case comparison.